Dockerfile

```diff
@@ -74,6 +75,7 @@ RUN if [ -n "${DOWNLOAD_USER}" ]; then PARAMS="-u ${DOWNLOAD_USER}"; fi && \
   mv /srv/downloads/${ARCHIVE_BASE_DIR} ${EXO_APP_DIR} && \
   chown -R ${EXO_USER}:${EXO_GROUP} ${EXO_APP_DIR} && \
   ln -s ${EXO_APP_DIR}/gatein/conf /etc/exo && \
+  mkdir -p ${EXO_CODEC_DIR} && chown ${EXO_USER}:${EXO_GROUP} ${EXO_CODEC_DIR} && \
```

Comment:
AFAIK the plf will create the codec folder in the startup

Reply:
Yes, it is. I have done some fixes on Meeds image which has problem with hsqldb (after container re-creation (no restart) data is deleted and new instance is available with older `gatein/data/files` files). After fixing this issue, codec file is tampered with every container re-creation. This issue was not detected because in our hosting infrastructure we used to bind this folder (as Docker volume) with correct folder permissions.

Reply:
ah ok, so the chown is the purpose not the location itself